content/backend/graphql-js/4-adding-a-database.md

```diff
@@ -7,189 +7,235 @@ answers: ["To increase performance of the server", "When using two APIs, the Gra
 correctAnswer: 2
 ---
 
-In this section, you're going to setup Prisma along with a connected database to be used by your GraphQL server.
+In this section, you're going to set up Prisma along with a connected database to be used by your GraphQL server. You'll be using [SQLite](https://www.sqlite.org/index.html) for the database.
 
-### Why Prisma
+## So, what is Prisma?
 
-By now, you already understand the basic mechanics of how GraphQL servers work under the hood - surprisingly simple right? That's part of the beauty of GraphQL, that it actually only follows a few very simple rules. The statically typed schema and the GraphQL engine that's resolving the queries inside the server are taking away major pain points commonly dealt with in API development.
+Prisma is an [open source](https://github.com/prisma/prisma) database toolkit that makes it easy for developers to reason about their data and how they access it, by providing a clean and type-safe API for submitting database queries.
 
-So, what's then the difficulty in building GraphQL servers?
+It mainly consists of three parts:
 
-Well, in real-world applications you're likely to encounter many scenarios where implementing the resolvers can become extremely complex. Especially because GraphQL queries can be nested multiple levels deep, the implementation often becomes tricky and can easily lead to performance problems.
-
-Most of the time, you also need to take care of many additional workflows such as authentication, authorization (permissions), pagination, filtering, realtime, integrating with 3rd-party services or legacy systems...
+- **Prisma Client**: An auto-generated and type-safe query builder for Node.js & TypeScript
+- **Prisma Migrate** (experimental): A declarative data modeling & migration system
+- **Prisma Studio** (experimental): A GUI to view and edit data in your database
 
-Typically, when implementing resolvers and connecting to the database, you have two options - both of which are not very compelling:
+In this tutorial, you will be seting everything up from scratch and taking full advantage of these three features. We want to get you building stuff right away, so explanations of Prisma concepts will be kept light but we have included links to [Prisma docs](https://www.prisma.io/docs/) in case you want to dive deeper on any particular concept.
 
-- Access the database directly (by writing SQL or using another NoSQL database API)
-- Use an [ORM](https://en.wikipedia.org/wiki/Object-relational_mapping) that provides an abstraction for your database and lets you access it directly from your programming language
+### Why Prisma?
```

Comment:
I really like the new version of this section, feels a lot more focussed and fits the flow of the tutorial 👍

Reply:
That's fantastic, such a relief to read that 😀🙌🏼